src/textnote.ts

```diff
@@ -13,12 +13,12 @@ export enum Justification {
 }
 
 export interface TextNoteStruct extends NoteStruct {
-  ignore_ticks?: boolean;
-  smooth?: boolean;
-  glyph?: string;
-  font?: FontInfo;
-  subscript?: string;
-  superscript?: string;
+  ignore_ticks: boolean;
+  smooth: boolean;
+  glyph: string;
+  font: FontInfo;
+  subscript: string;
+  superscript: string;
   text: string;
```

Comment:
Should we move `text` to the top of the TextNoteStruct declaration? It seems like the most important field here.

Reply:
I think that in many cases it is very difficult to figure out what is most important parameter. For exampla should we also put the scripts on the second position? In what order sub, super or super, sub?
I believe that the best approach would be to document them.

Reply:
I agree that the best approach will be to document each field. But we should leave this effort to a future PR. You and I can take on the various options object interfaces and add comments, especially where it is not clear what the field accomplishes.